classes/03class/exercises/c03-docker01/carloshz4/README.md

```diff
@@ -0,0 +1,18 @@
+# C03-Docker01
+
+## Docker 
+- [Dockerfile](Dockerfile)
+
+## Command Execution Output
+- Command to build the image:
+```
+docker build -t da-terraform:c03-docker-e01 .
+```
+
+- Command to run the image and its output.:
+```
+docker run -it da-terraform:c03-docker-e01
```

Comment:
It is missing the output of this command 😅

Reply:
Thanks Alan, you are going reviewing these exercises mate.  I've updated those missing commands.

Reply:
You're welcome 😁